src/components/curriculum/CurriculumReactBootcamp.js

```diff
@@ -47,43 +47,44 @@ const CurriculumReactBootcamp = ({
     type,
     isOpen,
   }
-  const trainingTime = '09:00 - 18:30'
   // TODO remove this DEPRECATED_ logic after July 2019
   const DEPRECATED_IS_JULY2019 =
     training.startDate && training.startDate === '2019-07-07T08:00:00.000Z'
   const meetupTime = DEPRECATED_IS_JULY2019 ? '16:30 - 18:30' : '18:30 - 21:00'
+  const trainingTime = ({ day }) =>
+    `${trainingDateByDay({ training, day })} ${
+      day !== 0 ? trainingTimings({ training }) : meetupTime
+    }`
+
   const firstHalf = (
     <React.Fragment>
       <H4>Course outline:</H4>
       <Section
         {...commonProps}
-        title="Evening pre-bootcamp (optional)"
+        title={`Evening pre-bootcamp`}
```

Comment:
@ric9176 isn't it optional anymore?

Reply:
This is not necessary from a user perspective as people who buy the course get further info where we specify that it's optional. On top of that, we really push people to attend that as we think it's necessary for them to do well on the course, so why say here that it's optional in bold?

Reply:
@ric9176 I agree 1) "people who buy the course get further info", 2) we really push people to attend that. That being said,  my concerns are:
A) When we run the training in London, not everyone attended the pre-training evening. If you were about to pay 2K for a 6-day training and you can't make it the first day (evening), what would you do? I guess: 1) ask us if it's possible to miss that evening (more work for Lena), 2) not buy the course (not good for us), 3) Buy the course and figure it out later. You are assuming that people will go option 3.
B) Not everyone reads all the messages we send. I think it's better to say something twice than once. If you pay 2K for training and then the first session you are along with people that paid 3.6 quid and you didn't read it in the post-purchase info, I'd be pissed off that I paid a few hundred times more for that session than the Meetup people.